### DISCUSSION TITLE
Welcome to the ChordMark forum!

### DISCUSSION THREAD

**Discussion by no-chris** [no-chris/chord-mark#470] [Announcements]
You can use this place to ask any question related to ChordMark and/or Chord Charts Studio.

Have fun!

**Comment by Rdeblock** [comment]
Is there a list of ChordMark parameters?

**Reply by no-chris** [reply]
in the source code directly:
https://github.com/no-chris/chord-mark/blob/master/packages/chord-mark/src/renderer/components/renderSong.js#L27
